test/engine/SpatialJoinAlgorithmsTest.cpp

```diff
@@ -780,8 +781,277 @@ TEST_P(SpatialJoinParamTest, computeResultSmallDatasetDifferentSizeChildren) {
       expectedMaxDist10000000_rows_diff, columnNames, false);
 }
 
-INSTANTIATE_TEST_SUITE_P(SpatialJoin, SpatialJoinParamTest, ::testing::Bool());
+INSTANTIATE_TEST_SUITE_P(
+    SpatialJoin, SpatialJoinParamTest,
+    ::testing::Values(SpatialJoin::Algorithm::Baseline,
+                      SpatialJoin::Algorithm::S2Geometry,
+                      SpatialJoin::Algorithm::BoundingBox));
 
 }  // end of Namespace computeResultTest
 
+namespace boundingBox {
+
+namespace bg = boost::geometry;
+namespace bgi = boost::geometry::index;
+
+typedef bg::model::point<double, 2, bg::cs::cartesian> point;
+typedef bg::model::box<point> box;
+typedef std::pair<point, size_t> value;
```

Comment:
Can't you reuse the typedefs/usings fromt the actual header file, so like
`using Point = SpatialJoinAlgorithms::Point` ;
or (even shorter)
`using SpatialJoinAlgorithms::Point`?